packages/tools/pie-icons-react/scripts/react-icons-gen.js

```diff
@@ -14,34 +14,57 @@ async function checkDirExists (directoryPath) {
         await fs.ensureDir(directoryPath);
         console.log(`Directory "${directoryPath}" exists.`);
     } catch (err) {
-        console.error(err)
+        console.error(err);
     }
 }
 
-const handleComponentName = name => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape
+const handleComponentName = (name) => name.replace(/\-(\d+)/, '$1'); // eslint-disable-line no-useless-escape
 
 // check that the /icons directory exists, if not create it
 checkDirExists(ICONS_DIR);
 
 // open a write stream to index.tsx
 const indexFile = fs.createWriteStream(
     filePath,
-    err => {
+    (err) => {
```

Comment:
does our linting really prefer parenthesis for single parameter functions?

Reply:
yep I believe so

Reply:
And yep – that's part of the new ruleset too. I have no strong feelings either way, but we went for the 'always put in brackets' option as it does keep a level of consistency.